opm/autodiff/SimulatorBase.hpp

```diff
@@ -191,6 +192,8 @@ namespace Opm
         RateConverterType rateConverter_;
         // Threshold pressures.
         std::vector<double> threshold_pressures_by_face_;
+        //Vertical flow performance tables
+        VFPProperties vfpProperties_;
```

Comment:
If we add this data member to the BlackoilModelBase class instead, could we not avoid changing the constructor interfaces? That would avoid introducing incompatibilities with downstream models (polymer, solvent etc.).